### PR TITLE
feat(pattern/fileuploader): add usage, code, specs, and design doc

### PR DESCRIPTION
**Pull request by marineFabien**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Other information

### CODE REVIEW

File: src/pages/Components/FileUploader/design.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+This is the section where you will find the ready to use symbols for your projects. You can find the different states of the file uploader.
+
+![fileuploader-sketch-menu2](fileuploader-sketch-menu2.png)
+
+- Default
+- With a single file
+- With a deleteable single file
+- With multiple files
+- With deleteable multiple files
```
[the remark this answers is not in the document]
Comment (marineFabien) [reply src/pages/Components/FileUploader/design.md]: you want to mean deletable ?

File: src/pages/Components/FileUploader/design.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### **File uploader :**
+
+This is the section where you will find the ready to use symbols for your projects. You can find the different states of the file uploader.
+
+![fileuploader-sketch-menu2](fileuploader-sketch-menu2.png)
+
+- Default
+- With a single file
+- With a deleteable single file
```
[the remark this answers is not in the document]
Comment (marineFabien) [reply src/pages/Components/FileUploader/design.md]: you want to mean deletable ?

File: src/pages/Components/FileUploader/design.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  <hintitem dont="true">
+  Never use the toolkit symbols in your project.
+  </hintitem>
+</hint>
+
+### **File uploader :**
+
+This is the section where you will find the ready to use symbols for your projects. You can find the different states of the file uploader.
+
+![fileuploader-sketch-menu2](fileuploader-sketch-menu2.png)
```
[the remark this answers is not in the document]
Comment (marineFabien) [reply src/pages/Components/FileUploader/design.md]: ??

File: src/pages/Components/FileUploader/design.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiagoux) [reply src/pages/Components/FileUploader/design.md]: actually... both are wrong. lol... let's rephrase.
With a ~deletable single file~ -> With single-file/multiple-files that can be deleted

File: src/pages/Components/FileUploader/design.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tiagoux) [reply src/pages/Components/FileUploader/design.md]: same typo on the image

File: src/pages/Components/FileUploader/design.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marineFabien) [reply src/pages/Components/FileUploader/design.md]: @nherwegh what dou you think about this term "deletable"

[the remark this answers is not in the document]
Comment (caqueste) [reply src/pages/Components/FileUploader/design.md]: I find the word !!! "Removable"

File: src/pages/Components/FileUploader/previews/file-uploader.preview.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<div class="example">
+    <div class="mc-fileuploader">
+        <input type="file" aria-label="Click to choose a file on your device" class="mc-fileuploader__input" id="import-file-id1">
```
[the remark this answers is not in the document]
Comment (marineFabien) [reply src/pages/Components/FileUploader/previews/file-uploader.preview.html]: Yes it should be :)

**Pull request merged**